docs/README.md

```diff
@@ -1,94 +1,26 @@
 # Introduction
 
-## What is Feast?
+Feast (**Fea**ture **St**ore) is an [open-source](https://github.com/feast-dev/feast) feature store that helps teams define, manage, validate, and serve machine learning features in production at scale. Whether you are training batch or real-time models, Feast reuses your existing data infrastructure to empower both training and inference workflows.
 
-Feast (**Fea**ture **St**ore) is an [open-source](https://github.com/feast-dev/feast) feature store that helps teams 
-operate production ML systems at scale by allowing them to define, manage, validate, and serve features for production 
-AI/ML. 
-
-Feast's feature store is composed of two foundational components: (1) an [offline store](getting-started/components/offline-store.md) 
-for historical feature extraction used in model training and an (2) [online store](getting-started/components/online-store.md) 
-for serving features at low-latency in production systems and applications.
-
-Feast is a configurable operational data system that re-uses existing infrastructure to manage and serve machine learning 
-features to realtime models. For more details, please review our [architecture](getting-started/architecture/overview.md).
-
-Concretely, Feast provides:
-
-* A Python SDK for programmatically defining features, entities, sources, and (optionally) transformations
-* A Python SDK for reading and writing features to configured offline and online data stores
-* An [optional feature server](reference/feature-servers/README.md) for reading and writing features (useful for non-python languages)
-* A [UI](reference/alpha-web-ui.md) for viewing and exploring information about features defined in the project
-* A [CLI tool](reference/feast-cli-commands.md) for viewing and updating feature information
-
-Feast allows ML platform teams to:
-
-* **Make features consistently available for training and low-latency serving** by managing an _offline store_ (to process historical data for scale-out batch scoring or model training), a low-latency _online store_ (to power real-time prediction)_,_ and a battle-tested _feature server_ (to serve pre-computed features online).
-* **Avoid data leakage** by generating point-in-time correct feature sets so data scientists can focus on feature engineering rather than debugging error-prone dataset joining logic. This ensures that future feature values do not leak to models during training.
-* **Decouple ML from data infrastructure** by providing a single data access layer that abstracts feature storage from feature retrieval, ensuring models remain portable as you move from training models to serving models, from batch models to real-time models, and from one data infra system to another.
-
-{% hint style="info" %}
-**Note:** Feast today primarily addresses _timestamped structured data_.
-{% endhint %}
+## Why Feast?
```

Comment:
I like having this here, probably we should put our most compelling offerings here.